easybuild/easyblocks/t/trinity.py

```diff
@@ -279,7 +286,16 @@ def install_step(self):
             else:
                 raise EasyBuildError("Don't know how to set TRINITY_COMPILER for %s compiler", comp_fam)
 
-            cmd = "make TRINITY_COMPILER=%s" % trinity_compiler
+            explicit_make_args = ''
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                explicit_make_args = 'all plugins'
+            # Delete CXXFLAGS CPPFLAGS and CFLAGS (interfere with plugins installation)
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                del os.environ['CXXFLAGS']
+                del os.environ['CPPFLAGS']
+                del os.environ['CFLAGS']
```

Comment:
@boegel: Maybe we have to consider compile everything with `Intel` compiler then?